boards/arm/ip_k66f/Kconfig.board

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2020 DENX Software Engineering GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_IP_K66F
+	bool "Segger IP-K66F"
+	depends on SOC_SERIES_KINETIS_K6X
+	select SOC_PART_NUMBER_MK66FN2M0VMD18
```

Comment:
Please intent by one tab instead of two spaces

Reply:
@MaureenHelm It was in tab, but just re-confirmed it. Thanks.